packages/grafana-ui/src/components/Button/Button.story.tsx

```diff
@@ -1,25 +1,41 @@
-import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { ComponentStory } from '@storybook/react';
 import React from 'react';
 
 import { ComponentSize } from '../../types/size';
 import { Card } from '../Card/Card';
 import { HorizontalGroup, VerticalGroup } from '../Layout/Layout';
 
-import { allButtonVariants, allButtonFills, Button } from './Button';
+import { allButtonVariants, allButtonFills, Button, ButtonProps } from './Button';
 import mdx from './Button.mdx';
 import { ButtonGroup } from './ButtonGroup';
 
-const meta: ComponentMeta<typeof Button> = {
+export default {
   title: 'Buttons/Button',
   component: Button,
   parameters: {
+    controls: { expanded: true },
     docs: {
       page: mdx,
     },
   },
+  argTypes: {
+    size: {
+      options: ['sm', 'md', 'lg'],
+    },
+    tooltip: {
+      table: {
+        disable: true,
+      },
+    },
+    tooltipPlacement: {
+      table: {
+        disable: true,
+      },
+    },
+  },
```

Comment:
Why did we disable these on the controls? I think we could let them be set to see how they work

Reply:
`tooltip` and `tooltipPlacement` props seems to be only used in `LinkButton` component

Reply:
Ah yes you are totally right!